docs/examples/expressions/expr_conditional_and_masking.ipynb

```diff
@@ -0,0 +1,386 @@
poZfaLlwjRNbDZboHupzXklx0jLDU9DJTMfMk/3H0NQNFWiBZIGLuoFy/GNRc6Z+9s7ym2PwzPnqU1xwRN8IvpEWStajHW3Y94fcGrUmtG89u4RYTts2Gw2DEPvYJ/fbByAKptxpObMfp7JJVNSQ5yjiVK85xAirRa2aUBrRUP6CxHfCSec8JOBE0eeOPLEkS+XI1/4kHr56AL1kWEtBFWomVArZ85ztTnr21sh4kLk9qOPud/f08ThXWAtmTAogxtw0u0XurdZQLVRtBAJtFZxCqVWnBXGMSKp4RwspbIsy7FzOpoOV+umxR7U9RFHbZlaCykFnNeHC8M5IaZAXTPDkLi7v+F7P/g+rzx6zNaPiAkUI5kQhx7KtYnDcXzgewpI6l+rtHa08ghoLrz/4+/yc1//5d5x9PNz9HVzR2839+dtNI4X6vMCrEezW3CYyrFj7B5xfawhUBrLshDdls10ztZ9CdeEenPPX/uVX+b3f+dfktzA1VtXfPWXvsm3//BPSesFZ+URY75kbgou8KXzt/nOH3+Xn3rndfZ5wUdHrIZXsLzl4smGGB5TSBRdGSUdNUFQaqOaILoyHK1Anv8+tVbM+e55RybEnsKx3X6Oi82W6eIV6uEpZfcRjgOeo7jaBUKgn39X+tjHGePYi/uMldqMUgNWDaxQWqY6uK/37NkjQVDnONTSPeackJ2weoEzj40DpSrqEtmMWHomtDRoBbwIYxqoOTPEkTSNaO037in1wntuG+J9Xz5Ae4Y00rdZpRli4BVCCjTXcN4z41GDaL5vzm5OoagnnPBZxIkjTxx54siXy5EvfEjdpg35zJDk0SVj68JmGHk0Tnzu8qxv2B12aF24GyPv3e9YzTNtExcX56j0okgpMW4nnAcLY4+Py426X5GlsRZFrLFfDDubcNuJzNFnbVGaKdVWxnFk4wTLDVB8j8wlewUHOTei96j1zkCk+4mlOJLN0bJnd7vgbYdshTEkLCshJAiKQ9icTSTvkJQYpwnfQ2vJy445Z+7nhVbh8mxgvv+QVAppmsi1YhIQ5/pJqY3gekTdc1Pi/HzzDyUER2vd/Fece7ig3fHvRiN5odTCOi/UJTBOV1y8Khw+eJfx4oKvfuunaXeKHzc8evvzPBHhx3d/iP8kc+62jM6z1MY4Br7w6hPOt9ODTka04K2SNLH74I7H/jH39ZYhJIYWONBvBmIOjxKdgMQHQfRzjYx30m1Plnu2m0SuMKQLCBumqxE9uyRfvUJZbmn3P8LcHbv1npFGq5W1GNGNcNx4TEPAfASBFDxyvHmXWilkonpGPKsszGVBg3UT5WBdg5UckjaMasi6suRKtYb4huRKsNj9/oqiNEYf8V4YRnfM3z4WxvHGWbVyaHu2cWA7nUNpzPuFuWRijJxv/vsiKQAAIABJREFUAqoHCIXtmHAx4Amcp4kheczV7nd3wgknfOZw4sgTR5448uVy5IvH/S6w3Zyz6C04RxUPIeBiYIiJIQrNn3N9f8eYBoZhwklkHEfGFJG4ZRgS3gvp6G9WvVBoUI1ijrYuUBVtrY8lcibnzLTdIt71/FvxOIN1KRTLjKlbOkRnxwu2+8ohxrxbGWLADUrwgj+KlocQcX5gbJ7YHGUuTFPibLslxC2oMsSAd4L3wuasJyfQlKaVYnB3mHl6cwctsswrvP8umzdncvb4EGmqOGeIBFotAA/ebt57TN2xALsvXtMCalirD13ig77IHb3vJB6j4bTnGlfj8RtfoC07nB/5/T/+HX78wUd8/+6O//A//c945jyvXV1xvS/UkBjFgWauzs9ZDis+jL1br0Z0Dt+Eicjt+0/Jl0rwnlWNahVvhgseU6MWwH/qAQfdF06tgBY46po2mzM224gP3XzY7ByXBvwwURzM9+9R1kLW9WiJcmDWilg3P55zxnkP3jOliZA8Yg6XhCkNNPXUJdPEiLGy5IVVC80ph7ZCNEwUwRPFod6R80JRBW/YMeatWWO7GTiPI947QgQflOh7ukkXr1faWrlbb8luos4rkyQqDZU+qrO2kpJn2gzEKeBjwBOZYiI4o+ER7/71BXbCCSf8ROPEkSeOPHHky+XIFz6kxjAiXrisxqHuITZq8MwK9/OM2YgopDCRBuP8MoIfenRbcAQ/Ys7wYt0+IkPzFTHDNSM4IAhtPaYw4BliRLxnySvb7dS95qyLd82M+6Wwn1dScIgJrvXuzh07sSByfM1ccdEf9SuhF4ELJDxTSGw3Gy7Pznny6DHp4oppGEAbh90dIv2VfJpG5t2eIo6K424/8/R+oeSMbEf46Ck/VWbgrHdOIcFxFBGPF+rzEU43NK4PHmpYQ0z7SEb7z+6cY5qmo4ZIui4oCjhBzVCB7B1+e8ZrX/06w9mH5H/yu5wPW758+Qruk2uuzGOHuXuhlYYSCF6IeIoKQzzOXQYjtoxvjfNxw90n13B2yUxjXfZI6DoTbd3oOaXu0ddaQbVvVKYUaFoxO45iDDabM7ZTwKzgXMDMkSSS/UB2W8L0hHn6kPn+E+b7G9T9kFZmnBOG0Eg+UlC8V0r0SAiYcwzjwHa7BdvCfWXdfci8LMz1nkMtzPNMtQdfYpJPeF/7MbZGWxUVo3lDAkzbSHAOXOnGxkPo1RAdzRUaQmkFi47iKnflmqdFeGW6YAoRQXEmjNOIBIUopBAJPoAK7thNF23PpVUnnHDCZwwnjjxx5IkjXy5HvvAhtRZFKwTXTYLdOHJoyu1a2UZAjCIOly5pNIYYMZ/YjgOD9+AiIsaaFywX5pzBF3COprUbx4rQPCAOnNAwmilBAqWU48afe9DQIFvysiCmBBOcGFI90QneOcSUhDBEz0YGgutWHVoU1U8tLLz3pGkkpEgcRvwQGWKitErwwt1dT8Aw51DnOVRllUhxA3MDq8KrKbHOO1K5YhhD755cN3nzTh4SJlo7bq7ZMd3BGta6qbJWo7VuXtyOnbKZ4ePA1aOLY0f2fNSRyVXxDPhtJFw15PyKu
```

Comment:
"We will use bitwise *OR* operation to build a mask representing the union of these regions. As the values in the mask are boolean, the bitwise `|`, `&` `^` operations can be used in similar fashion as their logical counterparts.\n",

Reply:
Done